### DISCUSSION TITLE
supabase cli - db commit function is disappeared

### DISCUSSION THREAD

**Discussion by magick93** [supabase/supabase#9365] [General]
Hello

I used to be able to run `supabase db commit <name>` and it would create a migration. 

Now this option has gone, and I dont see anything in the changelog about this change. 

Now, if I run `supabase migration new <name>` it just creates an empty file.

And `supabase db diff` prints to the console. 

Are these bugs? Or was there some announcement about these changes?

What is the recommended way to do a migration now?

**Comment by AaronDewes** [comment]
It's called `supabase db remote commit`: https://supabase.com/docs/reference/cli/usage#supabase-db-remote-commit, that still works on my side.

**Reply by magick93** [reply]
This is for the `remote` db, not local.

**Comment by soedirgo** [comment]
The new command is `supabase db diff --file <name>`: https://supabase.com/docs/reference/cli/usage#supabase-db-diff
